diff --git a/node_modules/telegram-test-api/telegramServer.js b/node_modules/telegram-test-api/telegramServer.js
index 8409c9a..378a86b 100644
--- a/node_modules/telegram-test-api/telegramServer.js
+++ b/node_modules/telegram-test-api/telegramServer.js
@@ -242,7 +242,7 @@ class TelegramServer extends EventEmitter {
    */
   deleteMessage(chatId, messageId) {
     const isMessageToDelete = (update) => (
-      update.message.chat.id === chatId && update.messageId === messageId
+      update.message.chat_id == chatId && update.messageId === messageId
     );
     const userUpdate = this.storage.userMessages.find(isMessageToDelete);
 
